rules/framework.bzl

```diff
@@ -515,6 +525,252 @@ def _attrs_for_split_slice(attrs_by_split_slices, split_slice_key):
     else:
         return attrs_by_split_slices[split_slice_key]
 
+def _bundle_dynamic_framework(ctx, avoid_deps):
+    """Packages this as dynamic framework
```

Comment:
Ah, so the apple_framework_packaging rule essentially reimplements ios_framework when link_dynamic is true.

Reply:
Yea, for now it calls into `rules_apple` bundling logic, which is somewhat similar to `ios_framework`. This is somewhat an implementation detail ATM. 

I'm a bit torn between doing this and using `ios_framework` - I like that this doesn't expand build file complexity or require us to make more macros. Possibly longer term we could propose a public API in rules apple:

```
bundling_support.bundle_dynamic_framework(binary=)
```